src/components/Search/SearchFiltersView.tsx

```diff
@@ -87,8 +88,9 @@ export default function SearchFiltersView({
       menuItems={filterItems}
       featureType={featureType}
       triggerIconClassName="absolute right-4 cursor-pointer max-h-[18px]"
+      onOpenChange={(open) => setIsOpen(open)}
       TriggerCustomRenderer={
-        <Tooltip tooltip={t('Search filter')}>
+        <Tooltip tooltip={t('Search filter')} hideTooltip={isOpen}>
```

Comment:
We have prop isTriggerClickable - doesn't it suit for you?

Reply:
@mikitabut , no `mouseOnly: isTriggerClickable` doesn't help with mouse over icon:
![image](https://github.com/epam/ai-dial-chat/assets/20163971/83053264-731e-41f0-afde-416b76bb8d0c)